strapi/strapi
# chore(content-releases): add permissions to content releases plugin

### What does it do?

Add permissions to the Content Releases plugin.

### How to test it?

You should have a valid engineering license and the feature flag `FEATURE_FLAG_CONTENT_RELEASES` enabled on your local environment.
1. As a superadmin user, go to edit the permissions of any role and go to the plugins tab, you should see a new section for "Content-releases" with permissions: Read, Create, Edit, Delete or Publish
2. If the user's role has permissions to read content releases they should see the Releases link on the sidebar
3. If the user's role has permissions to create content releases they should see the "Create new Release" button on the releases page
